static/app/components/replays/breadcrumbs/breadcrumbDescription.tsx

```diff
@@ -0,0 +1,96 @@
+import type {ReactNode} from 'react';
+import {isValidElement} from 'react';
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/core/button';
+import {Tooltip} from 'sentry/components/core/tooltip';
+import StructuredEventData from 'sentry/components/structuredEventData';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {ReplayFrame, WebVitalFrame} from 'sentry/utils/replays/types';
+import {isSpanFrame, isWebVitalFrame} from 'sentry/utils/replays/types';
+import type {OnExpandCallback} from 'sentry/views/replays/detail/useVirtualizedInspector';
+
+interface Props {
+  allowShowSnippet: boolean;
+  description: ReactNode;
+  frame: ReplayFrame | WebVitalFrame;
+  onClickViewHtml: (e: React.MouseEvent<HTMLButtonElement>) => void;
+  onInspectorExpanded: OnExpandCallback;
+  showSnippet: boolean;
+  className?: string;
+  expandPaths?: string[];
+}
+
+export function BreadcrumbDescription({
+  description,
+  allowShowSnippet,
+  showSnippet,
+  frame,
+  expandPaths,
+  onInspectorExpanded,
+  onClickViewHtml,
+}: Props) {
+  if (
+    typeof description !== 'string' &&
+    description !== undefined &&
+    isValidElement(description)
```

Comment:
I wanna double-check this. The original logic was:
```
typeof description === 'string' ||
      (description !== undefined && isValidElement(description))
```

I think the new phrasing will will kick `string` out, whereas before string was allowed in.